### PR TITLE
LPS-136356 Create Task node type component

### PR DESCRIPTION
**Pull request by naraandrade**
Story: [LPS-133990](https://issues.liferay.com/browse/LPS-133990)

### CODE REVIEW

**Comment by ramonmsp** [comment]
Just started reviewing :)

**Comment by jotascript** [comment]
Just started reviewing :)

File: modules/dxp/apps/portal-workflow/portal-workflow-instance-tracker-web/test/components/TaskNode.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		expect(container.querySelector('.lexicon-icon-check')).not.toBeTruthy();
+		expect(container.querySelector('.current-icon')).not.toBeTruthy();
+		expect(container.querySelector('.done-icon')).not.toBeTruthy();
+	});
+
+	it('Be rendered with live icon and current-icon class when it receives the current prop', () => {
+		const {container} = render(<TaskNode current />);
+
+		expect(container.querySelector('.lexicon-icon-live')).toBeTruthy();
+		expect(container.querySelector('.current-icon')).toBeTruthy();
```
[the remark this answers is not in the document]
Comment (naraandrade) [reply modules/dxp/apps/portal-workflow/portal-workflow-instance-tracker-web/test/components/TaskNode.js:32]: @ramonmsp I think it's ok :) I'll do it, thank you

**Comment by jotascript** [comment]
Just started reviewing :)

**Comment by naraandrade** [comment]
ci:forward

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:relevant

**Review by jotascript — APPROVED** [review]

**Pull request closed without merging**